ycast/radiobrowser.py

```diff
@@ -13,6 +13,7 @@
 SHOW_BROKEN_STATIONS = False
 ID_PREFIX = "RB"
 
+id_registry = {}
```

Comment:
What's that for? We can use `self.id`

Reply:
No we can't because the Access Error issue was caused by ID size limit to 15 chars in some radios. That's why we have to translate UUID (36 chars) to a fairly unique ID that will fit the limitation. Currently it is 12 chars, because with the prefix "RB_" it comes to exactly 15 chars.

Reply:
I understand.
Can we split this cache speed improvement off of the functionality fix? Would be nice for testing and traceability.

Reply:
Go for it, this version is now functional regarding addressed the UUID issue. Speed optimizations are way far on the schedule.